is-the-average-whole/main.js

```diff
@@ -0,0 +1,18 @@
+function isTheAverageWhole(numbersArray) {
+  var sum = 0;
+  var avg = 0;
+  for (var i = 0; i < numbersArray.length; i++) {
+    sum += numbersArray[i];
+  }
+  avg = (sum / numbersArray.length
```

Comment:
You are missing a closing parenthesis here. Though it may be a comfort, the parentheses are actually not necessary here, and using them when it is not necessary opens the door for little bugs like this 👍